src/sdk/model/__tests__/company.test.unit.ts

```diff
@@ -175,3 +175,50 @@ test('Properly submits request to get users with a specific role', async() => {
     });
   });
 });
+
+test('Properly submits request to get company logo', async() => {
+  const id = MockCompanyJson.uuid;
+  return Company.getCompany(id).then(async(company) => {
+    expect(Iland.getHttp().get).lastCalledWith(`/companies/${id}`);
+    expect(company.entityType).toBe('COMPANY');
+    return company.getLogo().then(async (logo) => {
+      expect(Iland.getHttp().get).lastCalledWith(
+        `/companies/${id}/logo`,
+        {
+          'headers': {'Accept': 'image/vnd.ilandcloud.api.v1.0+jpeg'},
```

Comment:
lets not hard-code the version in here because it will need to be updated every time we do a release.